db/install-db-resources.py

```diff
@@ -47,6 +48,18 @@ def create_database(conn, db_name):
         print("No database created due to empty parameter")
     return
 
+def remove_database(conn, db_name):
+    if db_name:
+        cursor = conn.cursor()
+        conn.autocommit = True
+        query = "DROP DATABASE {0};"
```

Comment:
Have you looked into using parameterized queries in Python with Postgres? Its obviously low risk for DB drop and create since they aren't runtime calls. I'd be happy to look into it so I can contribute some code to the project.

Reply:
@seushermsft I've been looking for the best practice in Python on how to do this. Any links?

Reply:
https://github.com/mfenniak/pg8000 says that we can use:

 `cursor.execute("SELECT array_prepend(:1, :2)", ( 500, [1, 2, 3, 4], ))` where `:1` and `:2` are the placeholders for the `500` and `[1, 2, 3, 4]` parameters.

That method requires that you specify a format type: `pg8000.paramstyle = "format"`

The interactive example page (https://pythonhosted.org/pg8000/quickstart.html#interactive-example) shows you can also use the more traditional templating format as long as you pass the true values into the execute method:

`cursor.execute(
        "INSERT INTO book (title) VALUES (%s), (%s) RETURNING id, title",
       ("Ender's Game", "Speaker for the Dead"))`

It looks like both of these will prevent injection attacks.

Reply:
I updated some other instances to use another method of formatting. You can't seem to parameterize identifiers in Postgres (tables names, extension names,  etc) so I'm going to leave these specific instances for now